### PR TITLE
Adding my profile information in a markdown file

### PR DESCRIPTION
**Pull request by mukatyz**
###Exercise 2 🤓🤓
--------------------------------------------------------------------------------------------------------------------------------------------
##Description

In this exercise we had create a file with our profile information in a markdown file

##Step by step
**1. Fork repository**

When you are in your github profile, you have clicking on the link https://github.com/undefined-academy/semana-1 and then click on the Fork button

**2. Clone the forked repository**

- Being in your forked repository, click on clone button, copy the SSH url.
- Open the terminal of your choice to manage git, in my case ubuntu.
- Type the `command git clone <ssh url>`

**3. Add a markdown file with your profile information**

- Open the route where you cloned the repository.
- Create a new file in the folder github-profiles and use your discord username as the file name (replacing the # character for -)
- Type your profile information in the file already created.
- Add your github profile link to the file

**4. Commit the changes**

- Open the terminal
- Type `git add .`
- Type `git commit -m "comment"`
- Type `git push`

**5. Create the pull request**

- Go to the first repository, where you did the Fork
- Click on Create new pull request
- Add a title and a description
- Click on Create pull request

### CODE REVIEW

File: github-profiles/mukatyz-2098/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+|Comando|Función|
+|-------|---------------------------------------------------------------|
+|ls|Lista los archivos de la carpeta actual, o de la ruta dada|
+|cd|Nos permite navegar entre carpetas|
+|rm-r|Nos permite eliminar una carpeta dada una ruta|
+|touch|Crea un archivo nuevo dada una ruta y/o nombre|
+|pwd|Nos nuestra la ruta absoluta del directorio actual, así podemos saber
+en dónde estamos ubicados.|
```
Comment (CardonaEA) [inline github-profiles/mukatyz-2098/README.md:34]: "en donde estamos ubicados" queda en una fila diferente.

File: github-profiles/mukatyz-2098/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        <tr> 
+            <th>rm="trash"</th>
+            <td>Se usa para mover un archivo a la papelera</td>
+        </tr>
+        
+    </table>
+<body>
+    
+</body>
+</html>
```
Comment (CardonaEA) [inline github-profiles/mukatyz-2098/index.html:67]: creo q s mejor colocar un linea al final del texto

File: github-profiles/mukatyz-2098/README.md (hunk 1 above)
Comment (mukatyz) [reply github-profiles/mukatyz-2098/README.md]: modificado, por favor revisa nuevamente

File: github-profiles/mukatyz-2098/index.html (hunk 2 above)
Comment (mukatyz) [reply github-profiles/mukatyz-2098/index.html]: modificado, por favor revisa nuevamente

**Review by CardonaEA — APPROVED** [review]

**Pull request closed without merging**